packages/itwinui-react/src/core/Table/Table.tsx

```diff
@@ -816,8 +823,41 @@ export const Table = <
     }
   });
 
+  const virtualizer = useVirtualizer({
+    count: page.length,
+    getScrollElement: () => tableElement ?? null,
+    estimateSize: () => 62, //Set to 62px since that is the default height of a table row.
```

Comment:
this estimate should probably differ based on the `density` prop

Reply:
Updated the estimate to vary on the `density` prop in this [commit](https://github.com/iTwin/iTwinUI/pull/2092/commits/0650ccddd1ec8800db5b6107ddcfd923ffe37095).